code/modules/mob/living/carbon/human/death.dm

```diff
@@ -101,13 +101,12 @@
 	SetJitter(0)
 	heart_attack = 0
 
+	for(var/obj/item/organ/O in organs) //these are external organs only: arms, legs, etc, internal organs are handled by carbon/death()

```

Comment:
then you'll have to make it loop through external, otherwise, this will do external *and* internal, then, when it calls `..()`, it'll run on the internal organs, again.